packages/components/src/combobox-control/index.js

```diff
@@ -46,7 +46,7 @@ function ComboboxControl( {
 	label,
 	options,
 	onChange,
-	onFilterValueChange,
+	onFilterValueChange = () => {},
```

Comment:
Let's use `noop` from `lodash` rather than inlining it here (that will follow the pattern of how we usually document intentional noop default props).